### ISSUE TITLE
Read The Docs overview diagrams are missing accessible descriptions or an alternative way to describe the content

### ISSUE THREAD

**Issue by fstrr** [intel/openfl#806]
**Describe the bug**
On Read The Docs overview page, there are diagrams that explain the difference between centralized learning and federated learning. These are complex diagrams that have no alt text, which means that screen reader users cannot hear a description of the image—instead they will hear part of the file name.

**To Reproduce**
This particular issue is relates to the Centralized and Federated Learning diagrams on the [overview page](https://openfl.readthedocs.io/en/latest/overview.html). However, it will also relate to any images in the documentation that don't have alternative text and/or aren't well described in the surrounding text.

**Expected behavior**
As these diagrams are complex, alt text would be a sub-optimal way to make their information accessible to assistive technology users. Instead, the information in the diagrams should be written out in semantic HTML. For example:
```html
<h2>Centralized Learning</h2>
<img alt="a diagram showing three hospitals sending data to a single data center, which then exports it to a model." src="centralized-learning-diagram">
<p>Standard machine learning approaches require centralizing the training data on one machine or in a datacenter. For example: multiple hospitals could send data into one data center that is used to train the learning model. The combined training data is then used to create the model.</p>
```

**Desktop (please complete the following information):**
 - All browsers and operating systems

**Smartphone (please complete the following information):**
 - All browsers and operating systems

**Additional context**
This is an accessibility issue.
